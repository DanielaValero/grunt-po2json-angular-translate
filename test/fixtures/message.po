msgid ""
msgstr ""
"Project-Id-Version: \n"
"POT-Creation-Date: \n"
"PO-Revision-Date: \n"
"Last-Translator: daniela Valero <danielavalero255@gmail.com\n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"


msgid "[:message/maintenance :title]"
msgstr "Maintenance"

msgid "[:message/maintenance :body]"
msgstr "There is currently some scheduled maintenance on our servers. \nPlease check back in a few minutes."
