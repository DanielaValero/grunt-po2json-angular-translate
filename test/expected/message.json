{
   "[:message/maintenance :title]": "Maintenance",
   "[:message/maintenance :body]": "There is currently some scheduled maintenance on our servers. \\nPlease check back in a few minutes."
}